governance/program/src/processor/process_update_program_metadata.rs

```diff
@@ -0,0 +1,67 @@
+//! Program state processor
+
+use borsh::BorshSerialize;
+use solana_program::{
+    account_info::{next_account_info, AccountInfo},
+    clock::Clock,
+    entrypoint::ProgramResult,
+    msg,
+    pubkey::Pubkey,
+    rent::Rent,
+    sysvar::Sysvar,
+};
+use spl_governance_tools::account::create_and_serialize_account_signed;
+
+use crate::state::{
+    enums::GovernanceAccountType,
+    program_metadata::{get_program_metadata_data, get_program_metadata_seeds, ProgramMetadata},
+};
+
+/// Processes UpdateProgramMetadata instruction
+pub fn process_update_program_metadata(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+
+    let program_metadata_info = next_account_info(account_info_iter)?; // 0
+    let payer_info = next_account_info(account_info_iter)?; // 1
+    let system_info = next_account_info(account_info_iter)?; // 2
+
+    let rent = Rent::get().unwrap();
+    let updated_at = Clock::get().unwrap().slot;
```

Comment:
nit: since the process function is returning a `ProgramResult`, these can be changed to 
```suggestion
    let rent = Rent::get()?;
    let updated_at = Clock::get()?.slot;
```
And that way, if something goes wrong, there's a nicer error than a panic

Reply:
that's cool, done